intel_extension_for_transformers/neural_chat/utils/model_utils.py

```diff
@@ -334,25 +124,33 @@ def import_deepspeed():
     if not is_deepspeed_available():
         raise ImportError(
             "This script requires deepspeed: `pip install"
-            " git+https://github.com/HabanaAI/DeepSpeed.git@1.10.0`."
+            " git+https://github.com/HabanaAI/DeepSpeed.git@1.11.0`."
         )
     # Initialize process(es) for DeepSpeed
     deepspeed.init_distributed(dist_backend="hccl")
     logger.info("DeepSpeed is enabled.")
 
 
-def init_deepspeed_inference(model, model_name_or_path, use_hpu_graphs):
+def init_deepspeed_inference(model, model_name_or_path, use_hpu_graphs, is_meta):
```

Comment:
why need this?

Reply:
it's from WangYi's change on workflow code.

Reply:
looks not good. can we fix? if not easy, can be in next pr